lib/controllers/friendship_controller.dart

```diff
@@ -0,0 +1,44 @@
+import 'dart:ffi';
+import 'package:flutter/cupertino.dart';
```

Comment:
```suggestion
import 'package:flutter/cupertino.dart';
```

import 'package:flutter/material.dart';